charts/argo-rollouts/README.md

```diff
@@ -1,19 +1,19 @@
-Argo Rollouts Chart
-=============
-A Helm chart for Argo Rollouts, progressive delivery for Kubernetes.
+# Argo Rollouts Chart
 
-Current chart version is `0.5.4`
+A Helm chart for Argo Rollouts, progressive delivery for Kubernetes.
 
 Source code can be found [here](https://github.com/argoproj/argo-rollouts)
 
 ## Additional Information
+
 This is a **community maintained** chart. This chart installs [argo-rollouts](https://argoproj.github.io/argo-rollouts/), progressive delivery for Kubernetes.
 
 The default installation is intended to be similar to the provided Argo Rollouts [releases](https://github.com/argoproj/argo-rollouts/releases).
 
 ## Prerequisites
 
 - Kubernetes 1.7+
+- Helm v3.0.0+
```

Comment:
Could you update the sample command in this file? The existing command uses `--name`, which is invalid for helpm3.